src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -353,7 +446,10 @@ definitions:
         description: A longer explanation of what this class is and what it can accomplish
   Tool:
     type: object
-    description: A tool (or described tool) describes one pairing of a tool as described in a descriptor file (which potentially describes multiple tools) and a Docker image.
+    description: >-
+      A tool (or described tool) is defined as a tuple of a descriptor file
+      (which potentially  consists of multiple files), a set of container image,
```

Comment:
extra space

Reply:
might be 'a set of container images' plural, similar to how the next line says 'a set of instructions'